hrui/js/src/hrui.js

```diff
@@ -188,9 +188,11 @@
    */
   function manipulateDOMOfInlineCustomData () {
     if ($('.Inline_Custom_Data').length) {
-      repositionInlineCustomDataBlockInPersonalDetailsTab();
+      repositionPersonalDetailsBlock('.Inline_Custom_Data', '.crm-summary-contactinfo-block');
     }
 
+    repositionPersonalDetailsBlock('.Extended_Demographics', '.Inline_Custom_Data');
```

Comment:
this line 
```js
if ($('.Inline_Custom_Data').length) {
```
implies that `.Inline_Custom_Data` might not exist. what then, in that case?